nomos-services/consensus/src/lib.rs

```diff
@@ -389,6 +396,21 @@ where
     DaPoolAdapter: MempoolAdapter<Item = DaPool::Item, Key = DaPool::Key> + Send + Sync + 'static,
     Storage: StorageBackend + Send + Sync + 'static,
 {
+    async fn handle_lifecycle_message(message: LifecycleMessage) -> bool {
```

Comment:
it's not clear from the signature what the bool in the return value means

Reply:
Yes, I have that itch as well. I will switch it for an enum.

Reply:
or maybe rename it something like `should_stop_service`

Reply:
> or maybe rename it something like `should_stop_service`

Yep, that would do as well. Thanks!